index.md

```diff
@@ -10,11 +10,17 @@ title: Home
 
 # {{ site.title }}
 
+
 ## Upcoming events
 
-* [LEAN meets MaRDI and OSCAR (5 - 6 Dec, 2024 at TU Berlin).](https://polymake.org/doku.php/workshops/lean_workshop1224)
-* [Computational Algebraic Geometry Workshop (18 - 22 Nov, 2024 at Durham University).](https://sites.google.com/view/durhamcompalggeom/home)
-* [MaRDI Annual Meeting 2024 (18 - 20 Nov, 2024 at Fraunhofer ITWM Kaiserslautern)](https://www.itwm.fraunhofer.de/de/messen-veranstaltungen/2024/2024_11_18_mardi-annual-meeting.html)
+{% assign sorted_conferences = site.data.conferences | sort: "end-date" | reverse %}
```

Comment:
```suggestion
{% assign sorted_conferences = site.data.conferences | sort: "end-date" %}
```

This will make earlier events show up closer to the top